src/stack.ts

```diff
@@ -182,7 +182,7 @@ export function stack(m: Mark | MarkDef, encoding: Encoding<string>): StackPrope
     // TBD: we should only enter this branch if there's also a "color" and maybe a group channel?
     const isPolar = isPolarPositionChannel(fieldChannel) || isPolarPositionChannel(dimensionChannel);
 
-    if (!hasSameDimensionAndStackedField && (isPolar ? dimensionChannel === fieldChannel || encoding['color'] : true)) {
+    if (!hasSameDimensionAndStackedField && (isPolar ? dimensionChannel === fieldChannel : true)) {
```

Comment:
Before I removed the color encoding check, the graph rendered like this: 

<img width="879" alt="image" src="https://github.com/user-attachments/assets/516eb254-b770-42e1-8dc0-a5ef20f22ad7" />